prow/cmd/autobump/bump.sh

```diff
@@ -177,6 +176,13 @@ list() {
   fi
 }
 
+split_on_commas() {
+  local IFS=,
+  local component_dirs=($1)
+  local suffix="${2:-/*.yaml}"
```

Comment:
This kind of seems overly specific for a function named "split on commas"

Reply:
> Do we already have an issue for converting this into a) a go program that b) contains the extra functionality provided by the k/t-i bumper?

Not that I am aware of. 

> This kind of seems overly specific for a function named "split on commas"

I broke it out into two separate functions.